### PR TITLE
Update core.py

### PR DESCRIPTION
**Pull request by srdey3993**
## What does this do?
Add support for GridPosition to specify size or position of a specific panel.

### CODE REVIEW

**Comment by bboreham** [comment]
Please tell us a bit more about what you are trying to achieve.

**Comment by srdey3993** [comment]
Hi,

I am trying to add support for GridPosition to specify size or position of
a specific panel.

Thanks,
Sudhanshu

File: grafanalib/core.py [hunk 1]
```diff
@@ -608,7 +618,7 @@ class Template(object):
         :param type: The template type, can be one of: query (default),
             interval, datasource, custom, constant, adhoc.
         :param hide: Hide this variable in the dashboard, can be one of:
-            SHOW (default), HIDE_LABEL, HIDE_VARIABLE
+            0 (default, no hide), 1 (hide label), 2 (hide variable)
```
Comment (bboreham) [inline grafanalib/core.py]: Why this change?

File: grafanalib/core.py [hunk 2]
```diff
@@ -853,7 +860,7 @@ def to_json_data(self):
 class Dashboard(object):
 
     title = attr.ib()
-    rows = attr.ib()
+    struct = attr.ib()
```
Comment (bboreham) [inline grafanalib/core.py]: This will break every existing user of Dashboard, right?
Is there a way to add what you want without breaking existing things?

File: grafanalib/core.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 """Low-level functions for building Grafana dashboards.
-
```
Comment (bboreham) [inline grafanalib/core.py]: Try to minimize irrelevant changes, like removing a blank line.  They make it harder for people to see what really changed.

**Comment by bboreham** [comment]
Hi, are you likely to come back to this?

**Comment by srdey3993** [comment]
No Bryan, Thanks.
